src/config/index.jsx

```diff
@@ -11,13 +11,13 @@ if (isDev) {
   ANDELA_API_BASE_URL = 'https://api.andela.com';
   BASE_URL = 'http://wire.andela.com:8080';
 } else if (isStaging) {
-  API_URL = 'https://wireapi-staging.herokuapp.com/api';
+  API_URL = 'https://api-staging.wire.herokuapp.com/api';
```

Comment:
Heroku does not allow this such URLs. Tried to rename our APi to `api-staging.wire` but it has a problem with the `.`.

Reply:
We will not be hosting the app on Heroku anymore. We will be using kubernetes. Another option is to use [custom env variables](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables) which will make the application more portable.